cmd/main.go

```diff
@@ -10,19 +11,28 @@ import (
 	"syscall"
 	"time"
 
-	"github.com/moycat/shiba/app"
-	"github.com/moycat/shiba/util"
 	log "github.com/sirupsen/logrus"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/moycat/shiba/app"
+	"github.com/moycat/shiba/util"
 )
 
 func main() {
 	if os.Geteuid() != 0 {
 		log.Fatal("shiba must be run as root")
 	}
-	config := parseConfig()
+	config := newConfig()
+	config.InitFlags(flag.CommandLine)
+	flag.Parse()
+	if err := config.Validate(); err != nil {
+		fmt.Println("node name is empty!")
+		fmt.Println()
```

Comment:
Actually I am considering sending this to stderr
```suggestion
		fmt.Fprintf(os.Stderr, "%v\n\n", err)
```